builtin/logical/pki/path_tidy.go

```diff
@@ -794,7 +819,7 @@ func (b *backend) startTidyOperation(req *logical.Request, config *tidyConfig) {
 			var revokedDeleted uint
 			var rebuildCRL bool
 
-			if config.CertStore {
+			if config.CertStore || config.InvalidCerts {
 				deleted, err := b.doTidyCertStore(ctx, req, logger, config)
 				if err != nil {
 					return err
```

Comment:
That also means I think on new line 836, you should also run the revocation store tidy if you are supposed to delete invalid certificates. They should be deleted from both places, I feel? My 2c!